src/NextjsRevalidation.ts

```diff
@@ -94,13 +108,94 @@ export class NextjsRevalidation extends Construct {
         },
       },
       // open-next revalidation-function
-      // see: https://github.com/serverless-stack/open-next/blob/274d446ed7e940cfbe7ce05a21108f4c854ee37a/README.md?plain=1#L65
+      // see: https://github.com/sst/open-next/blob/c2b05e3a5f82de40da1181e11c087265983c349d/docs/pages/advanced/architecture.mdx#L150
       entry: join(this.props.nextBuild.nextRevalidateFnDir, NEXTJS_BUILD_INDEX_FILE),
       handler: 'index.handler',
-      description: 'Next.js revalidation function',
+      description: 'Next.js queue revalidation function',
       timeout: Duration.seconds(30),
     });
     fn.addEventSource(new SqsEventSource(this.queue, { batchSize: 5 }));
     return fn;
   }
+
+  private createRevalidationTable() {
+    return new Table(this, 'RevalidationTable', {
+      partitionKey: { name: 'tag', type: AttributeType.STRING },
+      sortKey: { name: 'path', type: AttributeType.STRING },
+      pointInTimeRecovery: true,
```

Comment:
I think point in time recovery should be disabled by default to save cost. This is a cache, so cost of losing data is low. PITR costs $0.20/GB-month which isn't much. I could be persuaded otherwise though.

Reply:
I actually agree 100% - I tried to see if this was somehow included in free tier, but I don't think so. This is how SST has the table configured, but I guess they are less shy about the cost. $0.20 isn't much, but it's nice to not have to think about it and like you said, this is cache. 

I think if the data in this table is completely missing or corupted `revalidateTag` and `revalidatePath` will not work for SSG pages (based on  https://discord.com/channels/983865673656705025/1027265626085019769/1161575754727968801 ), BUT the site would still work and the table could be repopulated / redeployed easily it seems.